comps/llms/rayllm/README.md

```diff
@@ -0,0 +1,48 @@
+# RayLLM Endpoint Service
+
+[Ray](https://docs.ray.io/en/latest/serve/index.html) is an LLM serving solution that makes it easy to deploy and manage a variety of open source LLMs, built on [Ray Serve](https://docs.ray.io/en/latest/serve/index.html), has native support for autoscaling and multi-node deployments, which is easy to use for LLM inference serving on Intel Gaudi2 accelerators. The Intel Gaudi2 accelerator supports both training and inference for deep learning models in particular for LLMs. Please visit [Habana AI products](<(https://habana.ai/products)>) for more details.
+
+## Getting Started
+
+### Launch Ray Gaudi Service
+
+```bash
+bash ./serving/rayllm/launch_ray_service.sh
+```
+
+For gated models such as `LLAMA-2`, you will have to pass -e HUGGING_FACE_HUB_TOKEN=\<token\> to the docker run command above with a valid Hugging Face Hub read token.
+
+Please follow this link [huggingface token](https://huggingface.co/docs/hub/security-tokens) to get the access token and export `HUGGINGFACEHUB_API_TOKEN` environment with the token.
+
+```bash
+export HUGGINGFACEHUB_API_TOKEN=<token>
+```
+
+And then you can make requests like below to check the service status:
+
+```bash
+curl http://127.0.0.1::8080/v1/completions \
+  -H "Content-Type: application/json" \
+  -d '{
+  "model": <model_name>,
+  "prompt": "What is Deep Learning?",
+  "max_tokens": 32
+  }'
+```
+
+#### Customize Ray Gaudi Service
+
+The ./serving/rayllm/launch_ray_service.sh script accepts five parameters:
+
+- port_number: The port number assigned to the Ray Gaudi endpoint, with the default being 8080.
```

Comment:
better to highlight this, use `port_number`

Reply:
Updated it. Thanks.